spec/bolt/outputter/human_spec.rb

```diff
@@ -421,18 +424,60 @@
     it 'prints the inventory file path' do
       expect(File).to receive(:exist?).with(inventoryfile).and_return(true)
       outputter.print_targets(target_list, inventoryfile)
-      expect(output.string).to match(/Inventory file.*#{inventoryfile}/m)
+      expect(output.string).to match(/Inventory source.*#{inventoryfile}/m)
+    end
+
```

Comment:
It'd be great to include a smoke test for this in `spec/integration/inventory_spec`.

Reply:
Added for both `inventory show` and `group show`.